server/routes/user.py

```diff
@@ -74,7 +78,37 @@ def index():
     return "Data Commons </br> <a href='/user/auth/login'><button>Login</button></a>"
 
 
+@bp.route('/upload', methods=['POST'])
+@login_is_required
+def upload():
+    # TODO: changeg SECRETE_PROJECT to APP_PROJECT
+    # Upload files to GCS
+    user_id = session["google_id"]
+    project = current_app.config['SECRET_PROJECT']
+    bucket_name = project + '-resources'
```

Comment:
Is this a general resource bucket? Maybe a separate function in lib/gcs.py or lib/config.py instead of inline?